pipeline/external.go

```diff
@@ -33,7 +37,12 @@ func (e *external) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 		InputFile:     sourceFileUrl,
 		HLSOutputFile: job.TargetURL,
 		Profiles:      job.Profiles,
-		AutoMP4:       job.AutoMP4,
+		GenerateMP4: func(automp4 bool, duration float64) bool {
+			if automp4 && duration <= maxMP4OutDuration.Seconds() {
+				return true
+			}
+			return false
```

Comment:
I think this would make sense to be moved into pipeline/coordinator.go `StartUploadJob()`. Currently being in pipeline/external.go it means it still only applies for mediaconvert, there is a separate version of this `HandleStartUploadJob()` function in pipeline/mist.go

Reply:
Doh. I missed the Handler interface completely. Fixing...